lib/package_utils.rb

```diff
@@ -1,27 +1,12 @@
+# lib/package_utils.rb
+# Utility functions that take either a package object or a component of a package object as primary input.
 require 'json'
 require_relative 'const'
-require_relative 'crewlog'
 
 class PackageUtils
-  def self.load_json
-    return JSON.load_file(File.join(CREW_CONFIG_PATH, 'device.json'), symbolize_names: true).transform_values! { |val| val.is_a?(String) ? val.to_sym : val }
-  end
-
-  def self.save_json(json_object)
-    crewlog 'Saving device.json...'
-    begin
-      File.write File.join(CREW_CONFIG_PATH, 'device.json.tmp'), JSON.pretty_generate(JSON.parse(json_object.to_json))
-    rescue StandardError
-      puts 'Error writing updated packages json file!'.lightred
-      abort
-    end
-
-    # Copy over original if the write to the tmp file succeeds.
-    FileUtils.cp("#{CREW_CONFIG_PATH}/device.json.tmp", File.join(CREW_CONFIG_PATH, 'device.json')) && FileUtils.rm("#{CREW_CONFIG_PATH}/device.json.tmp")
-  end
-
   def self.installed?(pkg_name)
-    return load_json[:installed_packages].any? { |elem| elem[:name] == pkg_name }
+    device_json = JSON.load_file(File.join(CREW_CONFIG_PATH, 'device.json'))
+    return device_json['installed_packages'].any? { |elem| elem['name'] == pkg_name }
```

Comment:
Just use this?
```ruby
return ConvenienceFunctions.load_symbolized_json[:installed_packages].any? { |elem| elem['name'] == pkg_name }
```

Reply:
Just so our usage is synchronized. As a ruby newbie I was throughly confused by flitting between the symbolized and non-zymbolized usage.

Reply:
that's why i'm trying to swap entirely to strings :sweat_smile:

Reply:
Is there an advantage to using the string versions over the symbolized versions?

I'd be happy to just entirely use one or the other. Let's just not use a mix.

Thoughts @uberhacker @supechicken ?

Reply:
i'm moving to string versions just because it means we don't have to manually modify the returned array to use symbols-- there's no real reason otherwise, but theres no particular advantage to symbolized versions

Reply:
Agree that we need to be consistent.  I'm fine with all strings.  It's a bit less confusing.

Reply:
Can we switch everything over to string in this PR, and just modify fixup to redefine @device with a call to the load_json code so that it is no longer symbolized?

Reply:
Can we save that for a second PR? I'd be happy to do it, it just feels out of scope for this one.